net/HttpRequest.hpp

```diff
@@ -1201,7 +1201,7 @@ class Session final : public ProtocolHandlerInterface
     /// Returns the default timeout.
     static constexpr std::chrono::milliseconds getDefaultTimeout()
     {
-        return std::chrono::seconds(30);
+        return std::chrono::seconds(30); // FIXME?
```

Comment:
This is the default. There is nothing wrong with it. Please see `StorageConnectionManager::getHttpSession()` for how this is overrided.

Reply:
Thank you. 
Indeed, I simply marked all timeout values with FIXME to review them.
Goal is to allow us to make them configurable (clumsy word) @ runtime, 
i.e. at least via setter at runtime and/or via a config file (optional).
This is required to allow me to shape the unit tests, triggering timeouts and hence proving functionality.
You mentioned `WopiTestServer::configure()` and I will look into it.